### ISSUE TITLE
support for enableWhenExpression

### ISSUE THREAD

**Issue by rajesh-bandarupalli** [lhncbc/lforms-fhir-app#6]
Does SDC Smart App supports below extension?
http://build.fhir.org/ig/HL7/sdc/StructureDefinition-sdc-questionnaire-enableWhenExpression.html

I tried to use `enableWhenExpression` from this resource (http://community.fhir.org/t/how-do-i-use-enablewhenexpression/1983) and uploaded the questionnaire to SDC Smart App. But it doesn't work as expected.

**Comment by plynchnlm** [comment]
Not yet.  Support for that will come through the lforms package.  It is on our list.  (Sorry for the slow reply.  My GitHub notifications were getting filtered to the wrong folder.)  If this is a feature you critically need, let me know, and we can try to give it more priority.

**Comment by rajesh-bandarupalli** [comment]
I am just exploring this package and evaluating if it supports the use cases we have. By the way can i know if there is any plan in near future for upgrading this app to latest angular version?

**Comment by plynchnlm** [comment]
Yes, there is.  We are actually working on a web-component version, where the web components are created from Angular components.  That is under active development, but it is a complete re-write of the UI, and so it will likely be several more months before it is ready.
